Client/ViewModels/HomeViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using System.Collections.ObjectModel;
+using Client.Services;
+
+namespace Client.ViewModels
+{
+    public partial class HomeViewModel: BaseViewModel 
+    {
+        HomeService HomeService;
```

Comment:
This should be private readonly, and would prefer naming it something else, i.e. _homeService so it's obvious that it's injected from constructor